docs/serverless/rules/detections-logsdb-impact.asciidoc

```diff
@@ -5,7 +5,7 @@ LogsDB is enabled by default for Elastic serverless. This topic explains the imp
 
 With logsDB index mode, the original `_source` field is not stored in the index but can be reconstructed using {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`].
 
-When the `_source` is reconstructed, you can make {ref}/mapping-source-field.html#synthetic-source-modifications[modifications] close to the original data. However, there could be a mismatch between how users expect field values to be formatted.
+When the `_source` is reconstructed, {ref}/mapping-source-field.html#synthetic-source-modifications[modifications] are possible. Therefore, there could be a mismatch between how users expect field values to be formatted.
 
```

Comment:
Therefore, there could be a mismatch between how users expectations and how fields are formatted.

Reply:
Adding in https://github.com/elastic/security-docs/pull/6221/files#r1859257712.